opencti-platform/opencti-graphql/src/database/engine.js

```diff
@@ -602,6 +602,18 @@ const computePlatformMappings = () => {
           date_seen: { type: 'date' },
         },
       },
+      user_confidence_level: {
+        properties: {
+          max_confidence: { type: 'integer' },
+          overrides: {
+            type: 'nested',
+            properties: {
+              entity_type: { type: 'text' },
```

Comment:
I'm not sure type: 'text' is enough, I think we also need to define "keyword" field in entity_type, if we want to be able to do a term query on entity_type.

Reply:
indeed !